keyboards/gmmk/pro/rev1/ansi/rules.mk

```diff
@@ -13,3 +13,10 @@ AUDIO_ENABLE = no           # Audio output
 ENCODER_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = AW20216
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
```

Comment:
This is the default for f3, explicitly setting it does nothing

Reply:
Oh. Ok.

Where could I have found that information? For my own future reference.

Reply:
https://github.com/qmk/qmk_firmware/blob/feee0f5154465d925b0bada7f37b29167df4aab0/builddefs/common_features.mk#LL236-L240